src/rmc-compiler/rustc_codegen_rmc/src/codegen/function.rs

```diff
@@ -273,15 +277,52 @@ impl<'tcx> GotocCtx<'tcx> {
 
         self.proof_harnesses.push(harness);
     }
+    /// Update `self` (the goto context) to add the current function as a listed proof harness
+    fn handle_rmctool_unwind(&mut self, attribute_string: &str) {
+        /// clone parameter string for manipulation
+        let temp_string = attribute_string;
+
+        let vec_int: Vec<u32> = temp_string.chars().filter_map(|a| a.to_digit(10)).collect();
```

Comment:
Can you use parse instead? https://doc.rust-lang.org/std/primitive.str.html#method.parse